iOS/README.rst

```diff
@@ -131,10 +132,21 @@ In this invocation:
   Python, you can compile a python interpreter and then use that interpreter to
   run Python code. However, the binaries produced for iOS won't run on macOS, so
   you need to provide an external Python interpreter. This interpreter must be
-  the version as the Python that is being compiled.
-
-In practice, you will likely also need to specify the paths to iOS builds of the
-binary libraries that CPython depends on (XZ, BZip2, LibFFI and OpenSSL).
+  the *exact* same version as the Python that is being compiled.
+
+For a full CPython build, you also need to specify the paths to iOS builds of
+the binary libraries that CPython depends on (XZ, BZip2, LibFFI and OpenSSL).
+This can be done by defining the ``LIBLZMA_CFLAGS``, ``LIBLZMA_LIBS``,
+``BZIP2_CFLAGS``, ``BZIP2_LIBS``, ``LIBFFI_CFLAGS``, and ``LIBFFI_LIBS``
+environment variables, and the ``--with-openssl`` configure option. Versions of
+these libraries pre-compiled for iOS can be found in [this
+repository](https://github.com/beeware/cpython-apple-source-deps/releases).
+
+By default, Python will be compiled with an iOS deployment target (i.e., the
+minimum supported iOS version) of 12.0. To specify a different deployment
+target, provide the version number as part of the ``--host`` argument - for
+example, ``--host=arm64-apple-ios15.4-simulator`` would compile an ARM64
```

Comment:
It would make more sense to put this together with the `--host` instructions above.

Also, I know autoconf normalizes `arm64` to `aarch64`, but to reduce confusion we should at least be consistent within the README.

Reply:
I've normalised to arm64.